cluster/addons/calico-policy-controller/calico-node-daemonset.yaml

```diff
@@ -136,3 +151,68 @@ spec:
       tolerations:
         - key: "CriticalAddonsOnly"
           operator: "Exists"
+
+# Create all the required CustomResourceDefinitions needed
```

Comment:
Each CRD should be in its own yaml file in order to work with the add-on manager.